.github/workflows/run-build-and-acceptance-tests.yml

```diff
@@ -32,6 +34,8 @@ jobs:
 
             [1]: ${{ steps.vars.outputs.run-url }}
   go-lint:
+    if: |
+      github.event.label.name == 'ci/lint' || github.event.action != 'labeled'
```

Comment:
I'm not terribly keen on this being a label (which we're using primarily for metadata), having 2 different ways to execute tests could be confusing and would prefer for these to be slash commands